app-modules/service-management/src/Observers/ServiceRequestObserver.php

```diff
@@ -68,11 +74,20 @@ public function created(ServiceRequest $serviceRequest): void
         if ($serviceRequest->status?->classification === SystemServiceRequestClassification::Open) {
             $serviceRequest->respondent->notify(new SendEducatableServiceRequestOpenedNotification($serviceRequest));
         }
-    }
 
-    public function updating(ServiceRequest $serviceRequest): void
-    {
-        throw_if($serviceRequest->isDirty('service_request_number'), new ServiceRequestNumberUpdateAttemptException());
+        app(NotifyServiceRequestUsers::class)->execute(
+            $serviceRequest,
+            app(ServiceRequestCreated::class, ['serviceRequest' => $serviceRequest, 'channel' => EmailChannel::class]),
+            $serviceRequest->priority?->type->is_managers_service_request_created_email_enabled ?? false,
+            $serviceRequest->priority?->type->is_auditors_service_request_created_email_enabled ?? false,
+        );
+
+        app(NotifyServiceRequestUsers::class)->execute(
+            $serviceRequest,
+            app(ServiceRequestCreated::class, ['serviceRequest' => $serviceRequest, 'channel' => DatabaseChannel::class]),
```

Comment:
Why resolve the notification from the container? Unless we feel this will benefit us enough I would rather we new up so that we have proper type hinting and static analysis of the constructor arguments.

Reply:
I try and use the container everywhere so objects (even notifications) can be mocked or replaced in tests in the future if we need them, but I will change this now.

Reply:
Yeah, I think that approach makes sense. But I think for now it works for things like Notifications and Jobs to be instantiated.

Most of what we do is Unit testing or short scope Feature testing so most tests include ensuring a Job/notification WAS dispatched then a job/notification has it's own test.